.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
+}
```

Comment:
this change is unrelated to this PR

Reply:
I had to put the .vscode file in the git ignore file because in a PR I submitted earlier it was requested for another issue. Also it was in this PR that I added it. 

Please can you explain a way for me to solve that and also did I meet all the expectations?

Thank you

Reply:
When you add changes, you should add only the files related to the work you are doing rather than `git add -A`, in this way it makes sure you can have files present that are not added to a commit. 

You would need to remove this file, commit that, and then if you want to add it back be sure to not re add it to the commit again.

Reply:
I have removed the file and committed it, also the changes is in all the HTML files including the tests.